### DISCUSSION TITLE
Is 8kHz neural TTS possible?

### DISCUSSION THREAD

**Discussion by rachel-beeson-connex** [espnet/espnet#5383] [Q&A]
Hi all, I have been trying to develop a voice for TTS for the purposes of telephone communication. Currently I have had success with datasets sampled at 16 kHz. However, since the voices will be used on the phone, and because of latency concerns, I have been experimenting with 8kHz single speaker datasets. I have not been successful so far- most models only get to the "babbling" stage before they stop improving. Is there some inherent reason why neural TTS at 8kHz would not be possible? Is the signal too "weak"?

**Comment by kan-bayashi** [comment]
I think that is no problem at all when using 8k. Did you change the feature extraction setting to be the same length as 16k setting?
